.github/workflows/go-test.yaml

```
@@ -67,6 +67,11 @@ jobs:
           set -ex
           kubectl rook-ceph --context=$(kubectl config current-context) rados df
 
+      - name: Radosgw-admin create user
```

Let's use the lower case name of the tool everywhere
```suggestion
      - name: radosgw-admin create user
```